libs/user_extensions/src/TTAlpsSelections.cpp

```diff
@@ -17,8 +17,11 @@ void TTAlpsSelections::RegisterSignalLikeSelections(shared_ptr<CutFlowManager> c
   cutFlowManager->RegisterCut("nLooseMuonsOrDSAMuons");
 }
 
-bool TTAlpsSelections::PassesSignalLikeSelections(const shared_ptr<Event> event, shared_ptr<CutFlowManager> cutFlowManager) {
-  auto allMuons = asNanoEvent(event)->GetAllMuons(0.01);
+bool TTAlpsSelections::PassesSignalLikeSelections(const shared_ptr<Event> event, shared_ptr<CutFlowManager> cutFlowManager, string muonMatching) {
+  shared_ptr<PhysicsObjects> allMuons;
+  if(muonMatching == "DR") allMuons = asNanoEvent(event)->GetDRMatchedMuons(0.01);
```

Comment:
What if we don't want to apply any matching? Does it make sense to keep all muons without any replacements?

Reply:
I would suggest to rename the LooseMuons extra collection we have to LoosePATMuons and then also have an option to include the collection LooseMuons which would be LoosePATMuons + LooseDSAMuons, what do you think about me changing the naming?

Reply:
I agree we should probably rename `LooseMuons` -> `LoosePATMuons`. Still, I think it's a different issue - my question is if we should handle the case when `muonMatching` is `""`, and in this case it just returns all muons without any matching.